assets/src/dashboard/app/views/storyAnimTool/components.js

```diff
@@ -40,13 +40,15 @@ export const ActiveCard = styled.div(
     height: ${height}px;
     overflow: hidden;
 
-    ${Object.values(selectedElementIds).map(
-      (elementId) => `
-        [data-element-id='${elementId}'] {
-          border: 2px solid ${theme.colors.bluePrimary};
-        }
-      `
-    )}
+    ${Object.values(selectedElementIds)
+      .map(
+        (elementId) => `
+          [data-element-id='${elementId}'] {
+            border: 2px solid ${theme.colors.bluePrimary};
+          }
+        `
+      )
+      .join(' ')}
```

Comment:
This fixes a selection bug where more things were being highlighted than selected.

The bug:
![selection_bug](https://user-images.githubusercontent.com/40646372/83905212-23cd5600-a716-11ea-8a20-5c0577f92615.gif)

Now that it's fixed:
![selection_fix](https://user-images.githubusercontent.com/40646372/83905223-2af46400-a716-11ea-8e2a-95eb51fbcc2e.gif)